src/tasks/account/account_mixin.py

```
@@ -26,8 +26,16 @@ def __init__(self, *args, **kwargs):
             ),
         })
         self.default_config_group.update({
-            "多账户模式": ["多账户模式", "多账户独立配置", "账号列表"],
+            "多账户模式": ["多账户模式"],
         })
+        # 「多账户模式」开关：开启后展开显示「多账户独立配置」和「账号列表」两个子选项
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)